mibidata/mibi_image.py

```diff
@@ -439,77 +435,6 @@ def metadata(self):
         metadata_keys.extend(self._user_defined_attributes)
         return {key: getattr(self, key) for key in metadata_keys}
 
-    @staticmethod
-    def channels_is_list_of_tuples(channels):
```

Comment:
I liked the idea of having these defined as functions to make it clear what each is doing. Can we either reimplement the ones needed or document the code (e.g. line 319: `# Check if channels are tuples of mass and target`?

Reply:
I'm not opposed to these but think they belong in DPPA-732 as the scope of this PR should be tiff reader only and not extend to MibiImage changes.

Reply:
Sure, it can be done as part of DPPA-732.